tests/tools/test_dist_part.py

```diff
@@ -367,6 +332,13 @@ def test_pipeline_formats(graph_formats):
     _test_pipeline(4, 4, 4, graph_formats)
 
 
+"""
```

Comment:
remove them if not used any more

Reply:
This code will changed appropriately using PR #5310